### PR TITLE
[WAY-179] FEATURE** settings refactor/styling

### PR DESCRIPTION
**Pull request by dallascrichmond**
## Pull Request Standards

- [x] The title of the PR is accurate
- [x] The title includes the type of change [`HOTFIX`, `FEATURE`, `etc`]  
- [x] The PR title includes the ticket number in format of `[WAYF-###]`
- [x] Documentation is updated to reflect change

# Description

This PR includes the following proposed change(s):

- Created Accordion component
- Created Row button for settings
- Updated Settings CSS to meet style requirements
- Added Refresh setting that pulls in new app data if the user is online
- Added What's new button (currently routes to home view. need to add correct route)
- Fully Scalable

image

### CODE REVIEW

File: src/frontend-pwa/src/components/common/Accordion/accordion.styles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  margin: 0;
+  align-items: center;
+`;
+
+export const AccordionContent = styled.div<AccordionProps>`
+  display: ${(props) => (props.open ? 'block' : 'none')};
+  background-color: #ffffff;
+  padding: 5pt;
+  width: 100%;
+`;
```
Comment (LocalNewsTV) [inline src/frontend-pwa/src/components/common/Accordion/accordion.styles.ts:36]:
```suggestion
export const AccordionContent = styled.div<AccordionProps>`
  display: ${(props) => (props.open ? 'block' : 'none')};
  box-sizing: content-box;
  min-height: 3.5em;
  background-color: #ffffff;
  padding: 4pt;
  width: 100%;
    overflow-x: hidden;
`;
```

**Review by LocalNewsTV — COMMENTED** [review]
Some CSS tweaks for consideration

File: src/frontend-pwa/src/views/Settings/settings.styles.ts [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
+  padding: 50pt 0pt;
+  flex-direction: column;
+  max-height: 95%;
+  @media (min-width: ${mq.tablet}) {
+    width: 60%;
+  }
+  @media (min-width: ${mq.desktop}) {
+    width: 40%;
+  }
 `;
```
[the remark this answers is not in the document]
Comment (dallascrichmond) [reply src/frontend-pwa/src/views/Settings/settings.styles.ts]: @LocalNewsTV Done

File: src/frontend-pwa/src/views/Settings/settings.styles.ts [hunk 3]
```diff
@@ -1,72 +1,72 @@
...
-    max-width: 100vw;
-    padding: 1.5em;
-    padding-top: 100px;
-    padding-bottom: 100px;
+  &::-webkit-scrollbar-thumb {
+    background-color: rgba(0, 0, 0, 0.3);
+    border-radius: 4px;
+    background: none;
   }
 `;
```
[the remark this answers is not in the document]
Comment (dallascrichmond) [reply src/frontend-pwa/src/views/Settings/settings.styles.ts]: @LocalNewsTV Done

**Review by LocalNewsTV — APPROVED** [review]

**Pull request merged**